ngx-http-concat.php

```diff
@@ -176,15 +176,18 @@ function concat_get_path( $uri ) {
 		);
 
 		// The @charset rules must be on top of the output
-		if ( 0 === strpos( $buf, '@charset' ) ) {
+		if ( false !== strpos( $buf, '@charset' ) ) {
 			preg_replace_callback(
 				'/(?P<charset_rule>@charset\s+[\'"][^\'"]+[\'"];)/i',
 				function ( $match ) {
-					global $pre_output;
+					global $pre_output, $buf;
 
 					if ( 0 === strpos( $pre_output, '@charset' ) )
 						return '';
 
+					//strip $match[0] from $buf before adding $match[0] to beginning of $pre_output
+					$buf = str_replace($match[0], "", $buf);
```

Comment:
The `preg_replace_callback` will remove the match. This line is not required.

Reply:
@maxledoux The replaced value from the function `preg_replace_callback` isn't stored in the `$buf` variable. We can store the returned value. A separate replacement is not required. See similar example [here](https://github.com/Automattic/nginx-http-concat/pull/70/files#diff-c08378e3080a54702312abbe77d43a8dacb1c4928b49fc2531ea72b6ef26394eL199).

Reply:
I referenced [line number 199](https://github.com/Automattic/nginx-http-concat/blob/f2f9a38ed0946156559706f0d7daf6292cc52541/ngx-http-concat.php#L199), but it is not rendering in the UI.